.github/CHANGELOG.md

```diff
@@ -505,8 +527,8 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
   ```
 
   ```pycon
-  >>> tape = tape.expand(depth=2)
-  >>> print(tape.draw(wire_order=Wires(all_wires)))
+  >>> tape = tape.expand(depth=1)
```

Comment:
The original `tape = tape.expand(depth=2)` decomposes the `qml.MultiControlledX` operations and creates a way deeper circuit. Hence `depth=1` it the choice we'd like to showcase here.

Reply:
👍